plugins/org.yakindu.sct.doc.user/src/user-guide/sctunit.textile

```diff
@@ -5,370 +5,941 @@ p.
+
+The test assigns 18.0 to the statechart variable _p_, and performs one run-to-completion step. During this RTC, the state machine calls _getNeutronFlux(p)_ in order to evaluate the guard condition @[getNeutronFlux(p) >= 100.0]@. Since _p_ has a value of 18.0, the mocked operation returns 4.3, and the transition from *State_A* to *State_B* is not taken.
+
+After that, the test sets _p_ to 12.0 and executes another RTC. This time, the mocked operation returns 1000000000.0, the guard condition evaluates to _true_ and the state machine transitions to *State_B*.
+
+==</div>==
+
+The type of the default value specified in the _mock_ statement must match the return type of the mocked operation.
+
+It is also possible to make the default return value depending on the operation's parameters.
+
+bc. mock myOperation() returns (param + 1.0)
+
+Figure "&quot;Mock statement grammar&quot;":#sctunit_fig_mock_statement_grammar summarizes the structure of the _mock_ statement:
+
+p(#sctunit_fig_mock_statement_grammar). 
+
+!images/sctunit_grammar_020_mockreturnstatement.png(Mock statement grammar)!
+
+p=. Mock statement grammar
+
+==<!-- End sctunit_keyword_mock -->==
+
+
+h4(#sctunit_returning_an_operations_result). Returning an operation's result
+
+==<!-- Start sctunit_keyword_return -->==
+
+The _return_ statement terminates the execution of the current operation and returns the value of an expression to the caller. The type of the expression must match the operation's return type.
+
+==<div class="example">==
+
+Example:
+
+bc. 
+    operation getCircleArea (radius: real): real {
+        const pi: real = 3.141592654
+        return pi * pi * radius
+    }
+
+The _return_ statement evaluates the expression @pi * pi * radius@, i.e. the area of a circle with radius _radius_, and returns the result to the caller of the operation. .
+
+==</div>==
+
+Figure "&quot;Return statement grammar&quot;":#sctunit_fig_return_statement_grammar summarizes the structure of the _return_ statement:
+
+p(#sctunit_fig_return_statement_grammar). 
+
+!images/sctunit_grammar_020_returnstatement.png(Return statement grammar)!
+
+p=. Return statement grammar
+
+==<!-- End sctunit_keyword_return -->==
+
+
+
+
+h3(#sctunit_test_suites). Test suites
+
+A _test suite_ aggregates a set of "test classes":#sctunit_test_classes into a logical unit. It is contained in a file with a _.sctunit_ filename extension.
+
+==<div class="example">==
+
+Example:
+
+bc. testsuite MyTestSuite {
+    TestClassA,
+    TestClassB,
+    TestClassC
+}
+
+p. The test suite _MyTest_Suite_ comprises the test classes _TestClassA_, _TestClassB_, and _TestClassC_.
+
+==</div>==
+
+The nice thing about test suites is that you can run all tests of all the test classes at once. Right-click on the test suite file, say, _mytestsuite.sctunit_, and select _Run As → SCTUnit_ in the context menu. All tests in all test classes referenced in the test suite will be executed.
+
```

Comment:
Please add, that every referenced SCTUnit class in a SCTUnit suite must reference to statecharts with the same statechart type, ie if you have reference a SCTUnit class which reference to a CStatechart and another SCTUnit class which reference to a normal Default Statechart the SCTUnit Suite is not valid.